src/main/java/org/springframework/cloud/deployer/spi/scheduler/cloudfoundry/CloudFoundryAppScheduler.java

```diff
@@ -103,6 +106,13 @@ public void schedule(ScheduleRequest scheduleRequest) {
 		String scheduleName = scheduleRequest.getScheduleName();
 		logger.debug(String.format("Scheduling: %s", scheduleName));
 
+		if(scheduleName.length() > MAX_SCHEDULE_NAME_LENGTH) {
+			IllegalArgumentException illegalArgumentException = new IllegalArgumentException
+					(String.format("because the schedule name length exceeds %s characters. " +
```

Comment:
Will change to `The schedule could not be created because its name exceed the maximum allowed length of %s characters`.  I guess this should be in sync with the k8s one.